versioned_docs/version-8.7/self-managed/concepts/elasticsearch-privileges.md

```diff
@@ -3,32 +3,47 @@ id: elasticsearch-privileges
 title: "Elasticsearch privileges"
 ---
 
-If you implement Camunda 8 with Elasticsearch as a service provider, you must configure Elasticsearch with the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) in mind:
+If you implement Camunda 8 with Elasticsearch as a service provider, the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) may be required:
 
 ## Cluster privileges
 
-- `monitor` - necessary for health check
-- `manage_index_templates` to create and manage index schema on start up, if they don't already exist in Elasticsearch.
-- _Optional_ `manage_ilm` - required only when ILM is enabled
+:::note Running Elasticsearch with limited cluster privileges
+If an application cannot be granted cluster privileges, the schema manager can be run as a standalone application separate from the main application. In this setup, the single application does not need cluster privileges. To learn more, see [Elasticsearch without cluster privileges](./elasticsearch-without-cluster-privileges.md).
+:::
 
-To use the [backup feature](/self-managed/operational-guides/backup-restore/backup-and-restore.md), you must have snapshot privileges:
+- `monitor` - Required to check the Elasticsearch cluster health. This privilege provides read-only cluster operations permissions.
+- `manage_index_templates` - Creates the necessary index templates when Operate, Tasklist, and Optimize are started for the first time, or when updating to a newer version of Camunda 8. Once the index templates are created, you can stop the Component, remove this privilege, and then start Component again.
+- `manage_ilm` - _Required when index lifecycle management (ILM) is enabled._ Required to create the necessary ILM policies when Operate, Tasklist, and Optimize are started for the first time, or when updating to a newer version of Camunda 8. Once the ILM policies are created, you can stop the Component, remove this privilege, and then start the Component again.
 
-- `create_snapshot`
-- `monitor_snapshot`
+### Backup privileges
```

Comment:
Link to https://docs.camunda.io/docs/8.6/self-managed/concepts/elasticsearch-without-cluster-privileges/#standalone-backup-application